smart-doc/README.md

```diff
@@ -0,0 +1,176 @@
+# How to use these templates
+
+This README gives you a short introduction to our template files and how to use them.
+
+
+# Why these templates?
+
+These templates act as a blueprint for topic-oriented writing in DocBook.
+
+
+# Using these templates
+
+The templates in this directory are already contained in a directory structure that matches the 
+one of a real project. See the naming and directory conventions outlined below.
+
+If you are starting a topic-based authoring project from scratch, you can just copy this directory 
+tree and start adjusting file names (DC file, assembly and topics) and root IDs as outlined below.
+
+Once you have named the files accordingly, integrate them into the assembly file.
+
+Write the new content and adapt the IDs.
+
+In a final step, add suitable metadata information to your assembly file. Stick to the 
+conventions outlined below.
+
+
+# Templates for Smart Docs
+
+We provide five different types of information units:
+
+* `assembly.asm.xml`: Assembly (builds the actual "article" from the components listed below)
+* `task.xml`: Task (how to?)
+* `concept.xml`: Concept (what is?)
+* `reference.xml`: Reference (e.g., list of options, table with config files, default settings)
+* `glue.xml`: Combines texts or structures that do not fit into any of the other categories. 
+Typical glue topics include the intro section to your article, the "For more information" and the 
+"What's next" sections. Also use glue topics to add an additional layer of navigation to your 
+article. This is outlined in the `glue.xml` example file.
+
+
+## Naming and directory conventions
+
+When creating new articles and topics, please stick to the following conventions. We keep separate 
+directories for each topic type, for common files and articles.
+
+* Assembly/article files:
+
+      articles/<NAME>.asm.xml
+
+* Concept, reference, task, glue files:
+
+      concepts/<NAME>.xml
+      references/<NAME>.xml
+      tasks/<NAME>.xml
+      glues/<NAME>.xml
+
+* Common files (entities, gfdl, etc):
+
+      common/<NAME>.xml
+
+* Any topic's name consists of the following elements that are explained below:
+
+      <NAME> := <TOPIC>[-<SUBTOPIC]-<SNIPPET-TOPIC>-<TITLE>
+
+### DC file naming conventions
+
+The DC file name must match the assembly file's root ID.
+
+### File naming conventions
+
+Always start with the overarching topic. In doing so, ensure that articles belonging to the same topic can be easily spotted by someone looking for existing articles. If necessary, add a subtopic (append with `_`):
+```
```

Comment:
I think Markdown requires an additional empty line here.

Reply:
Hm, it seems to render fine, looking at the original file in https://github.com/SUSE/doc-modular/blob/main/templates/README.md